tgnnexplainer/xgraph/method/subgraphx_tg.py

```diff
@@ -274,7 +284,9 @@ def mcts_rollout(self, tree_node):
         return v
 
     def _select_expand_candidates(self, not_exist_children):
-        assert self.candidate_initial_weights is not None
```

Comment:
Actually these are initialized in the constructor of base explainer I believe
So there is no way this is none